src/Microsoft.Fx.Portability/ObjectModel/NuGetPackageInfo.cs

```diff
@@ -48,4 +49,34 @@ public override int GetHashCode()
             return _hashCode;
         }
     }
+
+    public class NuGetPackageInfoComparer : IComparer<NuGetPackageInfo>
+    {
+        public int Compare(NuGetPackageInfo x, NuGetPackageInfo y)
+        {
+            if (x == null)
+            {
+                if (y == null)
+                {
+                    return 0;
+                }
+                return -1;
+            }
+            else
+            {
+                if (y == null)
+                {
+                    return 1;
+                }
+                if (x.AssemblyInfo.Equals(y.AssemblyInfo, StringComparison.OrdinalIgnoreCase))
+                {
+                    return string.Compare(x.Target.FullName, y.Target.FullName, StringComparison.OrdinalIgnoreCase);
```

Comment:
Use the same compare logic as [NuGetPackageInfo.cs#L36-L37](https://github.com/alinapopa/dotnet-apiport/blob/145c4c1dd5d37e6b52559fe77f286b7e08811481/src/Microsoft.Fx.Portability/ObjectModel/NuGetPackageInfo.cs#L36-L37), and use `StringComparison.Ordinal`.

Reply:
Documentation for [`IComparer<T>`](https://msdn.microsoft.com/en-us/library/8ehhxeaf(v=vs.110).aspx#Remarks) suggests extending the abstract class, Comparer<T>.  Is it possible to make use of `Comparer<FrameworkName>.Default` for the comparison of Target rather than comparing the full names only?

Reply:
To make use of Comparer<T>.Default, the object, such as FrameworkName or NuGetPackageInfo has to implement IComparable<T>... An option is to change it to:

```csharp
public class NuGetPackageInfo : IComparable<T>
{
    //... implement IComparable<T> methods
    
    public override bool Equals(object o) {
        var compare = o as NuGetPackageInfo;
        return CompareTo(compare) == 0;
    }
}
```
and then use `Comparer<T>.Default` when trying to initialise a compare operation.

Reply:
Fixed.